tests/dhcp_relay/test_dhcp_pkt_recv.py

```diff
@@ -36,17 +37,27 @@ def check_dhcp_relay_feature_state(duthost):
 class Dhcpv6PktRecvBase:
 
     @pytest.fixture(scope="class")
-    def setup_teardown(self, duthost, tbinfo):
+    def setup_teardown(self, rand_selected_dut, tbinfo):
         disabled_host_interfaces = tbinfo['topo']['properties']['topology'].get('disabled_host_interfaces', [])
         ptf_indices = [interface for interface in tbinfo['topo']['properties']['topology'].get('host_interfaces', [])
                        if interface not in disabled_host_interfaces]
-        dut_intf_ptf_index = duthost.get_extended_minigraph_facts(tbinfo)['minigraph_ptf_indices']
+        dut_intf_ptf_index = rand_selected_dut.get_extended_minigraph_facts(tbinfo)['minigraph_ptf_indices']
+        if 'dualtor' in tbinfo['topo']['name']:
+            pattern = r'0.(\d+)'
+            ptf_indices = [int(re.findall(pattern, index)[0]) for index in ptf_indices if re.match(pattern, index)]
         yield ptf_indices, dut_intf_ptf_index
 
-    def test_dhcpv6_multicast_recv(self, duthost, ptfadapter, setup_teardown):
+    def test_dhcpv6_multicast_recv(
+        self,
+        rand_selected_dut,
+        setup_standby_ports_on_rand_unselected_tor,
```

Comment:
In addition to this, We need `toggle_all_simulator_ports_to_rand_selected_tor` fixture (needed for `active-standby` topologies)

Reply:
Also other fixtures in this module (Ex: setup_teardown_acl) need to have `rand_selected_dut` fixture.